src/main/java/org/openbase/bco/bcozy/model/InfluxDBHandler.java

```diff
@@ -0,0 +1,337 @@
+package org.openbase.bco.bcozy.model;
+
+import org.openbase.jul.exception.CouldNotPerformException;
+import org.influxdata.client.*;
+import org.influxdata.query.FluxRecord;
+import org.influxdata.query.FluxTable;
+import org.openbase.jul.exception.printer.ExceptionPrinter;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class    InfluxDBHandler {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(InfluxDBHandler.class);
+    private static final String INFLUXDB_BUCKET_DEFAULT = "bco-persistence";
+    private static final String INFLUXDB_URL_DEFAULT = "http://localhost:9999";
```

Comment:
cant find the issue

Reply:
what about an issue number?

Reply:
than please create one ;)

Reply:
![image](https://user-images.githubusercontent.com/38458182/60433454-2ae60d00-9c05-11e9-81fe-81ae1aa19c39.png)

**existing** issue? I do not even know what he full scope if the referenced **existing** issue is.

Reply:
Sorry, I thought @DavidWein94 already created an issue for moving the influx db into the bco.dal.remote api since he is already working a couple of week on this issue. However, I will clarify this tomorrow in a short meeting.

Reply:
Okay, that sounds like he will most likely create an issue tomorrow. Then I can link it.

Reply:
Already linked by @DavidWein94